src/libraries/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/SAM/SAMUtils.cs

```diff
@@ -197,7 +195,7 @@ internal static string PAPIQueryToRegexString(string papiString)
 
                         default:
                             sb.Append(@"\\");
```

Comment:
This call can be also changed to `Append(char)`, because its logic is simpler.

Reply:
the `@` in `@"\\"` escapes `\`, so this isn't equivalent to `'\'`, it is a 2 char string.

see [verbatim - Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim)

however, somewhat related to my comment, this `default` should not even be reachable.  because this else block should be reachable

Reply:
Note the `escapeMode` variable is toggled inside the switch block @ line [164](https://github.com/dotnet/runtime/pull/32686/files#diff-6c172f905d958380968c8b25ec610189L164).